draft-ietf-quic-http.md

```diff
@@ -508,113 +508,110 @@ HEADERS frames can only be sent on request / push streams.
-    weight between 1 and 256.
+  : An optional unsigned 8-bit integer representing a priority weight for the
+    prioritized element (see {{!RFC7540}}, Section 5.3). Add one to the value
+    to obtain a weight between 1 and 256.  When absent, indicates the resource
+    should be delivered all at once or not at all.
+
+  Priority:
+  : An optional unsigned 8-bit integer representing a strict priority for the
+    prioritized element (see {{!RFC7540}}, Section 5.3).  When absent, this
+    value is 16.  An element with a higher priority should be delivered before
+    an element of lower priority.
+
 
 The values for the Prioritized Element Type ({{prioritized-element-types}}) and
 Element Dependency Type ({{element-dependency-types}}) imply the interpretation
 of the associated Element ID fields.
 
-| PT Bits | Type Description | Prioritized Element ID Contents |
-| ------- | ---------------- | ------------------------------- |
-| 00      | Request stream   | Stream ID                       |
-| 01      | Push stream      | Push ID                         |
-| 10      | Placeholder      | Placeholder ID                  |
-| 11      | Current stream   | Absent                          |
+| P Bit | Type Description | Prioritized Element ID Contents |
+| ----- | ---------------- | ------------------------------- |
+| 0     | Current stream   | Absent                          |
+| 0     | Push stream      | Push ID                         |
+| 1     | Placeholder      | Placeholder ID                  |
 {: #prioritized-element-types title="Prioritized Element Types"}
 
-| DT Bits | Type Description | Element Dependency ID Contents |
-| ------- | ---------------- | ------------------------------ |
-| 00      | Request stream   | Stream ID                      |
-| 01      | Push stream      | Push ID                        |
-| 10      | Placeholder      | Placeholder ID                 |
-| 11      | Root of the tree | Absent                         |
+| D  Bit | Type Description | Placeholder ID Contents |
+| ------ | ---------------- | ----------------------- |
+| 0      | Root of the tree | Absent                  |
+| 1      | Placeholder      | Placeholder ID          |
 {: #element-dependency-types title="Element Dependency Types"}
 
 Note that unlike in {{!RFC7540}}, the root of the tree cannot be referenced
-using a Stream ID of 0, as in QUIC stream 0 carries a valid HTTP request.  The
-root of the tree cannot be reprioritized.
+using 0, so the root of the tree cannot be reprioritized.
 
 The PRIORITY frame can express relationships which might not be permitted based
 on the stream on which it is sent or its position in the stream. These
 situations MUST be treated as a connection error of type HTTP_MALFORMED_FRAME.
 The following situations are examples of invalid PRIORITY frames:
 
 - A PRIORITY frame sent on a request stream with the Prioritized Element Type
```

Comment:
Since the list is one bullet point long with this change, you could probably roll this back into prose

Reply:
I added back another one, but it's still a short list.  I believe one more will end up being added, so keeping the structure for now to minimize diffs.